internal/connect/zypper.go

```diff
@@ -32,6 +33,11 @@ const (
 	zypperInfoReposSkipped    = 106 // Some repository had to be disabled temporarily because it failed to refresh
 )
 
+var (
+	//go:embed zypper-restore.tmpl
+	restoreTemplate string
```

Comment:
this should be in `zypperbackup.go` where it's actually used

Reply:
done